docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1878,12 +1878,16 @@ is false.
     ```
 
 `<FILE>` is the name of the private key file used to
-[sign Privacy Marker Transactions](../../HowTo/Use-Privacy/Sign-Privacy-Marker-Transactions.md). If
-you do not specify this option, Besu signs each transaction with a different randomly generated
-key.
+[sign Privacy Marker Transactions](../../HowTo/Use-Privacy/Sign-Privacy-Marker-Transactions.md).
 
-If using [account permissioning] and privacy, you must specify a private key file and include the
-corresponding public key in the accounts allowlist.
+You must specify this option if:
+
+* Using a privacy network where you pay gas. Also, the associated account must contain adequate funds.
+* Using [account permissioning] and privacy. You must include the corresponding public key in the
+    accounts allowlist.
+
+If you do not specify this option (for example, in a free gas network), Besu signs each transaction
+with a different randomly generated key.
```

Comment:
while you're going it would be good to make clear here https://besu.hyperledger.org/en/stable/Reference/CLI/CLI-Syntax/#privacy-marker-transaction-signing-key-file that it's perfectly acceptable to use the same as the --node-private-key-file
 this can be the nodekey itself or something else you'd like to use to identify who signed the marker tx.

Reply:
Done.